File: src/main/java/com/github/nsc/de/shake/parser/node/logical/LogicalBiggerNode.java
package com.github.nsc.de.shake.parser.node.logical;

import com.github.nsc.de.shake.parser.node.ValuedNode;

public class LogicalBiggerNode extends LogicalCompareNode {

    public LogicalBiggerNode(ValuedNode left, ValuedNode right) { super(left, right); }

    @Override
    public String getOperator() {
        return ">";
    }

}

File: parser/src/commonMain/kotlin/io/github/shakelang/shake/parser/node/logical/LogicalBiggerNode.kt
package io.github.shakelang.shake.parser.node.logical

import io.github.shakelang.shake.parser.node.ValuedNode
import io.github.shakelang.parseutils.characters.position.PositionMap

class LogicalBiggerNode(map: PositionMap, left: ValuedNode, right: ValuedNode) : LogicalCompareNode(map, left, right) {
    override val operator: String
        get() = ">"

    override fun toJson(): Map<String, *> = mapOf("name" to "LogicalBiggerNode", "left" to left, "right" to right)
}